### ISSUE TITLE
Don't assume decimals for pDAI conversion

### ISSUE THREAD

**Issue by vrde** [leapdao/plasma-burner-wallet#246]
The amount of decimals for an ERC20 token depends on the value stored in the smart contract. In the burner wallet we just assume that any ERC20 token has the same amount of decimals as Ether.

https://github.com/leapdao/plasma-burner-wallet/blob/75e3e3888bb284574b4c19bcc6f52c608459fdc0/src/App.js#L386

**Comment by TimDaub** [comment]
Good point. I looked into the web3 implementation of `fromWei` but unfortunately it just allows conversion based on ethereum unit names, while a conversion based on the decimal number would make much more sense to me personally.

I wish there was a function like `fromWei(<num>, 18)`...
